src/odemis/driver/hamamatsurx.py

```diff
@@ -1308,12 +1390,12 @@ def AcqStop(self, timeout=1):
         """Stops the currently running acquisition.
         :parameter timeout: (0.001<= float <=60) The timeout value (in s)
         until this command should wait for an acquisition to end.
-        :return: (float) timeout (in s)"""
+        :return: (float) rtimeout (in s)"""
```

Comment:
In the docstring, you can leave "timeout", as it's clearer what you're talking about.